### ISSUE TITLE
As a developer, I don't want to have to share internal paths on the server with users to support their ability to refer to previously posted data

### ISSUE THREAD

**Issue by epag** [NOAA-OWP/wres#153]
---

Author Name: **Hank** (Hank)
Original Redmine Issue: 124390, https://vlab.noaa.gov/redmine/issues/124390
Original Date: 2023-12-22

---

See #121242. When a user posts data, the response will include the absolute path of the posted data in @content-location@. James mentioned that we could use the @md5@ blob to identify the data. When I pointed out that the file name, without the path, could serve that purpose, he responded:

> Asking a user to nominate their evaluation as "data direct" when they have the relevant paths identities and the data already exists is a bit clunky, it should "just work" without any special designation as data direct if the server can see the data and the data is identified properly. 

In other words, if the user modifies the declaration to point to the data blobs, however that is done, the tasker should be able to figure it out and prepare a proper declaration for the workers. 

This ticket can be resolved when a user is able to reuse previously posted data without having to provide a complete path to the created files, and the WRES stops returning that path in the HTTP response.

Hank

**Comment by epag** [comment]
---

Original Redmine Comment
Author Name: **James** (James)
Original Date: 2024-06-28T11:49:47Z

---

( Leaking paths is also bad for security. )
